### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ioxbank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,17 +85,19 @@ Want to contribute to the list? Read the [contributors guide](CONTRIBUTING.md)!
...
 * [Reddit#bitshares](https://reddit.com/r/bitshares) - Reddit sub-reddit for BitShares.
 * Telegram:
-  * [BitShares DAC](https://t.me/BitSharesDAC) - General discussions.
-  * [BitShares Group](https://t.me/BitSharesGroup) - Another group for general discussions.
-  * [BitShares News](https://t.me/BitSharesNews) - BitShares News channel
+  * [BitShares Group](https://t.me/BitSharesGroup) - Group for general discussions.
+  * [BitShares DAC](https://t.me/BitSharesDAC) - Another group for general discussions.
+  * [BitShares News](https://t.me/BitSharesNews) - BitShares News channel.
   * [BitShares Mobile](https://t.me/btsplusplus) - Focus on BitShares mobile app.
+  * [BitShares Mobile](https://t.me/BitSharesWallet) - Focus on BitShares reference wallet.
```
Comment (abitmore) [inline README.md:98]: Please update the name.

Comment (ioBanker) [reply README.md:98]: Sorry; didn't get which name that you're referring to.

Comment (abitmore) [reply README.md:98]: > Sorry; didn't get which name that you're referring to.

The group name "BitShares Mobile".

Comment (ioxbank) [reply README.md]: Done, thanks.

**Review by abitmore — APPROVED** [review]

**Pull request merged**